VTEX - Subscriptions API v3.json

```diff
@@ -3622,14 +3622,14 @@
                         "type": "object",
                         "description": "Object with [payment method](https://help.vtex.com/en/tutorial/difference-between-payment-methods-and-payment-conditions--3azJenhGFyUy2gsocms42Q) information.",
                         "required": [
-                            "paymentSystem"
+                            "paymentSystem",
+                            "paymentAccountId"
                         ],
                         "properties": {
                             "paymentAccountId": {
                                 "type": "string",
-                                "description": "Information about the customer profile, such as their credit card number. To obtain that information, use the endpoint [Get client profile by email](https://developers.vtex.com/docs/api-reference/checkout-api#get-/api/checkout/pub/profiles).",
-                                "example": "340357032569595",
-                                "nullable": true
+                                "description": "Information about the customer profile. To obtain that information, use the endpoint [Get client profile by email](https://developers.vtex.com/docs/api-reference/checkout-api#get-/api/checkout/pub/profiles) to find the `accountId`.",
```

Comment:
This example doesn't look like what I would expect from the description. Exactly what information is represented here? What is the "payment account" that names the field?